### DISCUSSION TITLE
树图GraphinTreeData不能设置边的样式吗？

### DISCUSSION THREAD

**Discussion by sawanoutsu** [antvis/Graphin#484] [Q&A]
树图GraphinTreeData不能设置边的样式吗？我想设置树图的边的颜色，但是GraphinTreeData结构里就没有边相关的字段

**Comment by pomelo-nwu** [comment]
@sawanoutsu  Graphin 对 TreeData 支持的不好，这个我们将在 G6 5.0 和 Graphin 3.0 中做升级，目前 2.0 基本上没太多好办法
